src/ray/core_worker/core_worker.h

```diff
@@ -684,8 +684,8 @@ class CoreWorker : public rpc::CoreWorkerServiceHandler {
   /// \param[in] IDs of the objects to wait for.
   /// \param[in] num_objects Number of objects that should appear.
   /// \param[in] timeout_ms Timeout in milliseconds, wait infinitely if it's negative.
-  /// \param[out] results A bitset that indicates each object has appeared or not.
-  /// \return Status.
+  /// \param[out] results A vector of booleans that indicates each object has appeared or
+  /// not. \return Status.
```

Comment:
nit: formatting, `\return` should stay in the newline

Reply:
oops, sorry. The linter updates it automatically. I will update it.

Reply:
follow up: https://github.com/ray-project/ray/pull/49441#issue-2759124683